runtime/themes/dark_plus.toml

```diff
@@ -6,7 +6,7 @@
 "type" = { fg = "type" }
 "type.builtin" = { fg = "type" }
 "type.enum.variant" = { fg = "constant" }
-"constructor" = { fg = "constant" }
+"constructor" = { fg = "type" }
```

Comment:
\cc @kirawi to confirm this

Reply:
Hmm, I think that the usage of `@constructor` has become somewhat conflated with `@type` as in many highlight queries it applies to all identifiers with `^[A-Z]`. I'm fine with this change.

Reply:
The recent commits we made to this PR fixed up the usage of `@constructor` so it is no longer conflated with `@type` in python at least. I suspect originally that was the main motivation for @Zeddicus414 wanting to change this. For style reasons, maybe the change here is still a good idea. I just thought I'd point this out.